libs/output-mapping/src/Storage/StoragePreparer.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\OutputMapping\Storage;
+
+use Keboola\OutputMapping\Mapping\MappingFromProcessedConfiguration;
+use Keboola\OutputMapping\Mapping\MappingStorageSources;
+use Keboola\OutputMapping\Storage\BucketCreator;
+use Keboola\OutputMapping\Storage\TableDataModifier;
+use Keboola\OutputMapping\Storage\TableInfo;
+use Keboola\OutputMapping\Storage\TableStructureModifier;
+use Keboola\OutputMapping\SystemMetadata;
+use Keboola\StorageApi\ClientException;
+use Keboola\StorageApiBranch\ClientWrapper;
+use Psr\Log\LoggerInterface;
+
+class StoragePreparer
+{
+    public function __construct(readonly private ClientWrapper $clientWrapper, readonly private LoggerInterface $logger)
```

Comment:
na toto nejsou testy  - což je celkem důležiý - že se opravdu zavolá table structure updater a tabledata modifiert
a podle mě by bylo v pohodě dát si je do ctoru a namockovat je - bude to víc vypadat jako DI i když DI sem jen tak nedostaneme